### PR TITLE
change scene_indices, fix visualisation issue

### PR DESCRIPTION
**Pull request by lucabergamini**
address https://github.com/lyft/l5kit/issues/39 by:
- changing default val indices list (-1 for all)
- looking for first valid frames with agents in visualisation notebook

### CODE REVIEW

File: examples/agent_motion_prediction/agent_motion_prediction.ipynb [hunk 1]
```diff
@@ -47,7 +47,7 @@
    "outputs": [],
    "source": [
     "# set env variable for data\n",
-    "os.environ[\"L5KIT_DATA_FOLDER\"] = \"PATH_TO_YOUR_DATA\"\n",
+    "os.environ[\"L5KIT_DATA_FOLDER\"] = \"/tmp/l5kit_data\"\n",
```
Comment (oliver-scheel) [inline examples/agent_motion_prediction/agent_motion_prediction.ipynb:50]: why this hardcoded path now?

Comment (yye-lyft) [inline examples/agent_motion_prediction/agent_motion_prediction.ipynb:50]: I would not recommend user to save data in `/tmp` folder, which means you need to re-download data after reboot. I would either let user to decide `PATH_TO_YOUR_DATA ` or some permanent storing place

File: examples/agent_motion_prediction/agent_motion_prediction.ipynb [hunk 2]
```diff
@@ -327,7 +338,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.3"
+   "version": "3.6.8"
```
Comment (yye-lyft) [inline examples/agent_motion_prediction/agent_motion_prediction.ipynb:341]: What's the reason roll back to 3.6.8?

**Comment by oliver-scheel** [comment]
LGTM

File: examples/agent_motion_prediction/agent_motion_prediction.ipynb (hunk 1 above)
Comment (lucabergamini) [reply examples/agent_motion_prediction/agent_motion_prediction.ipynb:50]: sorry, forgot to reset it :)

Comment (lucabergamini) [reply examples/agent_motion_prediction/agent_motion_prediction.ipynb:50]: I know, I just forgot to reset it :)

File: examples/agent_motion_prediction/agent_motion_prediction.ipynb (hunk 2 above)
Comment (lucabergamini) [reply examples/agent_motion_prediction/agent_motion_prediction.ipynb:341]: this is just jupyter keeping track of the python version sadly...

**Pull request merged**